src/analyzer/protocol/ssl/spicy/SSL.spicy

```diff
@@ -0,0 +1,1901 @@
+
+  return True;
+}
+
+# public type TLSMessage = unit {
+#  %context = Share;
+#
+#  m: Message(False);
+# };
+#
+# public type DTLSMessage = unit {
+#  %context = Share;
+#
+#  m: Message(True);
+# };
+
+public type Message = unit {
+  %context = Share;
+
+  sink handshakesink;
+  sink alertsink;
+  var record_version: uint16;
+  var dtls: bool = False;
+
+  on %init {
+    self.handshakesink.connect(new Handshake(self, self.context()));
+    self.alertsink.connect(new Alert(self.context()));
+    # print "top-level init";
+  }
+
+  fragment : RecordFragmentChoice(self.handshakesink, self.alertsink, self, self.context())[];
+  # : bytes &eod if ( self.context().skipping );
+};
+
+function is_dtls_version(version: uint16) : bool {
+  if ( version == DTLSv10 || version == DTLSv12 )
+    return True;
+
+  return False;
+}
+
+type RecordFragmentChoice = unit(handshakesink: sink&, alertsink: sink&, inout msg: Message, inout sh: Share) {
+  content_type: uint8; # &convert=ContentType($$);
+  version: uint16;
+
+  switch ( is_dtls_version(self.version) ) {
+    True -> dtlsfragment : DTLSRecordFragment(self.content_type, handshakesink, alertsink, msg, sh);
+    False -> tlsfragment : TLSRecordFragment(self.content_type, handshakesink, alertsink, msg, sh);
+  };
+
+  on content_type {
+    # print "Content type", self.content_type;
+  }
+  on version {
+    msg.record_version = self.version;
+    if ( is_dtls_version(self.version) ) {
+      msg.dtls = True;
+    }
+  }
+};
+
+
+
+type TLSRecordFragment = unit(content_type: uint8, handshakesink: sink& , alertsink: sink&, inout msg: Message, inout sh: Share) {
+  record: PlaintextRecord(content_type, handshakesink, alertsink, msg, sh);
+};
+
+type DTLSRecordFragment = unit(content_type: uint8, handshakesink: sink&, alertsink: sink&, inout msg: Message, inout sh: Share) {
+# the epoch signalizes that a changecipherspec message has been received. Hence, everything with
+# an epoch > 0 should be encrypted
+  epoch: uint16;
+  sequence_number: bytes &size=6; # uint48
+  record: PlaintextRecord(content_type, handshakesink, alertsink, msg, sh);
+};
+
+type PlaintextRecord = unit(content_type: uint8, handshakesink: sink&, alertsink: sink&, inout msg: Message, inout sh: Share) {
+  length: uint16;
+  var encrypted: bool;
+  # convenient triggers to hang stuff in the evt file from. Two of them for event ordering :)
+  trigger_zero: void;
+  trigger_one: void;
+  switch ( ContentType(content_type) ) {
+    ContentType::handshake -> : bytes &size=self.length -> handshakesink;
+    ContentType::application_data -> {
+      switch ( self.encrypted ) {
+        False -> appdata: bytes &size=self.length;
+        True -> cryptdata: bytes &size=self.length;
+      };
+    }
+    ContentType::change_cipher_spec -> ccs : bytes &size=self.length;
+    ContentType::heartbeat -> hn: Heartbeat(sh, self.length);
+    ContentType::alert -> : bytes &size=self.length -> alertsink;
+    * -> unhandled : bytes &size=self.length;
```

Comment:
You can use `skip` here to simply jump over `self.length` bytes,

```suggestion
    * -> unhandled: skip bytes &size=self.length;
```

There are few more instances where `skip` can be used.